static/docs/get-started/metrics.md

```diff
@@ -1,6 +1,6 @@
 # Experiment Metrics
 
-The last stage we would like to add to our pipeline is its the evaluation. Data
+As a final stage we would like to add an evaluation part to our pipeline. Data
```

Comment:
Thanks for noticing the typos! Definitely better. But let's avoid the "part" term and keep "stage" related to "pipeline". How about "Finally, we would like to add an evaluation stage to our pipeline"?

Reply:
I'l address this.